packages/subiquity_client/lib/src/server.dart

```diff
@@ -113,6 +113,7 @@ abstract class SubiquityServer {
     for (var i = 0; i < 10; ++i) {
       try {
         await client.send(request);
+        await client.close();
```

Comment:
Wouldn't it be clearer to have this call outside of the for loop?

Reply:
Thanks! Yes, and correct too